src/shared/scrapers/USA/MA/index.js

```diff
@@ -132,6 +132,10 @@ const scraper = {
       let sumDeaths = 0;
 
       data.features.forEach(item => {
+        if (item.attributes.County === 'TOTALS') {
```

Comment:
To avoid a potential regression, would be possible to make this check case insensitive?